plugins/generator-1.21.1/neoforge-1.21.1/procedures/utils/projectiles/throwable.java.ftl

```diff
@@ -0,0 +1,9 @@
+private static Projectile initThrowableProjectile(Projectile entityToSpawn, Entity shooter, Vec3 acceleration) {
+	if (shooter != null)
+		entityToSpawn.setOwner(shooter);

```

Comment:
Not sure if null check is needed as it is also done in projectile code

![image](https://github.com/user-attachments/assets/b9d73db6-3067-4f28-861d-6d74d290e448)

Reply:
So I can remove all the checks like `input$shooter != "null"` from those templates?

Reply:
You likely still need `(input$shooter == "null"),` otherwise 

`new ${generator.map(field$projectile, "projectiles", 0)}(${generator.map(field$projectile, "projectiles", 1)}, projectileLevel)`

will be used when acceleration is zero, even if the shooter is not null which is not correct.

Btw you asked if you need to remove all checks like

`input$shooter != "null"`

but I was not able to find them in the changed code anyways

Reply:
Any comparison between shooter input and `"null"`

Reply:
> You likely still need `(input$shooter == "null"),` otherwise 
> 
> `new ${generator.map(field$projectile, "projectiles", 0)}(${generator.map(field$projectile, "projectiles", 1)}, projectileLevel)`
> 
> will be used when acceleration is zero, even if the shooter is not null which is not correct.